install/migrations/update_10.0.x_to_10.1.0/entity.php

```diff
@@ -44,3 +44,53 @@
 if (!$DB->fieldExists("glpi_entities", "inquest_mandatory_comment", false)) {
     $migration->addField('glpi_entities', 'inquest_mandatory_comment', "int NOT NULL DEFAULT '0'", ['after' => 'inquest_default_rate']);
 }
+
+$fields = [
+    'is_contact_autoupdate',
+    'is_user_autoupdate',
+    'is_group_autoupdate',
+    'is_location_autoupdate',
+    'is_contact_autoclean',
+    'is_user_autoclean',
+    'is_group_autoclean',
+    'is_location_autoclean'
+];
+$config = \Config::getConfigurationValues('core');
+foreach ($fields as $field) {
+    if (!$DB->fieldExists("glpi_entities", $field, false)) {
+        $migration->addField(
+            'glpi_entities',
+            $field,
+            "tinyint NOT NULL DEFAULT '" . \Entity::CONFIG_PARENT . "'"
```

Comment:
We should always use hardcoded values/names in migrations, to prevent a change in migration behaviour in case of value change in the future.
```suggestion
            "tinyint NOT NULL DEFAULT '-2'"
```